examples/create_upload_tutorial.py

```diff
@@ -30,8 +30,8 @@
 data = np.concatenate((X, y.reshape((-1, 1))), axis=1)
 attribute_names = list(attribute_names)
 attributes = [
-    (attribute_name, 'REAL') for attribute_name in attribute_names
-] + [('class', 'REAL')]
+                 (attribute_name, 'REAL') for attribute_name in attribute_names
+             ] + [('class', 'INTEGER')]
```

Comment:
Could you please rename this to 'target' as it is actually a regression dataset?

Reply:
http://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_breast_cancer.html 
It is classification